### ISSUE TITLE
Light Theme: Add Portfolios link to top nav bar

### ISSUE THREAD

**Issue by meetinghouse** [meetinghouse/cms#111]
The top nav bar currently has three links: Home, About Page, and Contact Page. Can you please add a link to Portfolios (which will display a drop-down of all the sub-portfolios when clicked, similar to the left nav bar) to the top nav bar?

Here is how it looks currently:
http://steeplejacksworld.corbettresearchgroup.com/
![screenshot 2015-04-16 at 1 30 15 pm](https://cloud.githubusercontent.com/assets/5215893/7187151/bcc3302a-e43c-11e4-97e0-0c7796216638.png)

**Comment by acavanagh** [comment]
This is updated in the blog branch, need to push it to an example site for review

**Comment by acavanagh** [comment]
hi - you can see this at steeplejacks world now.

**Comment by meetinghouse** [comment]
That looks great. Before it is deployed across sites, is it possible to move it to second position from the left, so the order of the top nav is the same as the left nav?

**Comment by serundeputy** [comment]
`PR` #160 moving portfolios top nav position; @acavanagh have a look :smile:

**Comment by meetinghouse** [comment]
Hey, folks. I noticed today that the portfolio link in the top menu isn't reflecting across all the sites I thought were fully updated. For instance, moultoncustomdoor is not showing this update. Can this change please be pushed to all the client sites?

Frankly, I am afraid to run deploy code because it may bork a site. Is it safe for me to simply hit deploy without making any code changes? Should this type of change get deployed automatically at this point, or is that in the future?

**Comment by acavanagh** [comment]
I just updated the deploy script on
MOULTONCUSTOMDOOR.CORBETTRESEARCHGROUP.COM so that you shouldn't get the
timeout error any more.  If you use that script on any of the recently
updated sites you should be fine.

**Comment by meetinghouse** [comment]
I copied that code and ran it for two sites, but wasn't successful in getting either one updated to have the portfolios link in the top nav.

I ran it for bronzedoorrestoration and the site it still up, but the top nav doesn't have the new link in it.

I ran it for thayer52 and now the site is not accessible. This site wasn't as updated as bronzedoorrestoration, so obviously I should not have tried this code for it. I thought it was perhaps far enough along to be deployed, though. I don't have a way of knowing which ones are okay and which one aren't unfortunately. So that one is currently down.

**Comment by acavanagh** [comment]
I just checked on these, thayer52 appeared to be working and updated.
Bronzedoors wasn't updated, upon further checking there was a typo in the
name of the site - it was set up
as bronzedoorrestoration.corbettresearchgroupcom - note the lack of a .
before the .com.  I changed the deploy script to work with the typo for
that site.  It's fully updated now.

**Comment by acavanagh** [comment]
can this be closed?

**Comment by meetinghouse** [comment]
OK, so the typo thing is making me feel dumb. I do not know why I couldn't get to thayer52 when I submitted this issue, because it is now fine. It was giving me an error before, but whatever. Don't care, since it has resolved. Maybe I needed to give it another minute or two. Thanks.

And there is a sweet flavor of sucesssssss, as I am able to deploy code across all sites! I'm closing this issue and happily deploying updated code across the board now. Thank you so much.
